src/main/resources/swagger.api/schemas/entityTypeColumn.json

```diff
@@ -20,6 +20,10 @@
         "isIdColumn": {
           "description": "Indicates if the column is part of the unique identifier for this entity type",
           "type": "boolean"
+        },
+        "sourceView": {
+          "description": "Source view for this column",
+          "type": "string"
```

Comment:
We may not need/want this, but this could provide an easy way to match a field with its source, to make dealing with aliases easier.